### ISSUE TITLE
why MNIST Acc is very highly in FGSM MNIST

### ISSUE THREAD

**Issue by muyuuuu** [Harry24k/adversarial-attacks-pytorch#75]
In short, I use squeezenet to train a model in mnist dataset which acc is 99.71.

Then, I use `torchattacks.FGSM(model)` to generate attack data, acc is 97.38, is it normally ?

**Comment by Harry24k** [comment]
Could you check the normalization of the images? It should be in a range of [0, 1].
Please attach the file to reproduce.

**Comment by muyuuuu** [comment]
```py
def get_origin_data(data_names, name=None):
    assert name is not None, "Please Input Dataset Name"
    data_name = ""
    for i in data_names:
        data_name += i
        data_name += " "
    assert name in data_names, "Only Support DataSet: [{}]".format(data_name)

    data_path = "./origin_data"

    if name == "mnist":
        transform = torchvision.transforms.Compose([
            torchvision.transforms.RandomCrop(32, padding=4),
            torchvision.transforms.ToTensor(),
            torchvision.transforms.Normalize((0.1307, ), (0.3081, ))
        ])
        train_dataset = torchvision.datasets.MNIST(root=data_path,
                                                   train=True,
                                                   download=True,
                                                   transform=transform)
        test_dataset = torchvision.datasets.MNIST(root=data_path,
                                                  train=False,
                                                  download=True,
                                                  transform=transform)


trainset, testset = get_origin_data(data_names=data_names, name="mnist")
trainloader = torch.utils.data.DataLoader(trainset,
                                          batch_size=batch_size,
                                          shuffle=False)
testloader = torch.utils.data.DataLoader(testset,
                                         batch_size=batch_size,
                                         shuffle=False)
```

**Comment by Harry24k** [comment]
`torchvision.transforms.Normalize((0.1307, ), (0.3081, ))` should be deleted.
Please refer to https://github.com/Harry24k/adversarial-attacks-pytorch/issues/69

**Comment by muyuuuu** [comment]
> Please refer to #69

Thank U~

I have another question after read #69 , should I delete all the `norm` in dataloader, such as `cifar10` and `cifar100` by other attack method?

**Comment by muyuuuu** [comment]
emmm, here is other results in mnist. Only CW attack works. all the method has used `torchvision.transforms.Normalize((0.1307, ), (0.3081, ))`


| model  | fgsm  | cw2   | pgd   | autoattack | jitter | sparsefool |
| ----------- | ----- | ----- | ----- | ---------- | ------ | ---------- |
| sequeezenet | 97.28 | 11.35 | 93.33 |            | 94.23  |            |

**Comment by muyuuuu** [comment]
![image](https://user-images.githubusercontent.com/43681138/170964397-63fc3a6f-45fc-4818-848e-b826ebfff6e0.png)

the visualizatin of cw attack, maybe this is the reason of attacking works.

**Comment by Harry24k** [comment]
This is because CW uses the L2-norm and there is no upper limit for the distance between the adversarial image and the original image.
Don't use Normalize in transforms.

> All images should be scaled to [0, 1] with transform[to.Tensor()] before used in attacks. To make it easy to use adversarial attacks, a reverse-normalization is not included in the attack process. To apply an input normalization, please add a normalization layer to the model. Please refer to [code](https://github.com/Harry24k/adversarial-attacks-pytorch/blob/master/demos/White%20Box%20Attack%20(ImageNet).ipynb) or [nbviewer](https://nbviewer.jupyter.org/github/Harry24k/adversarial-attacks-pytorch/blob/master/demos/White%20Box%20Attack%20%28ImageNet%29.ipynb).
